crates/sui/tests/checkpoints_tests.rs

```diff
@@ -239,10 +239,12 @@ async fn checkpoint_with_shared_objects() {
 
     // Make a few test transactions.
     let total_transactions = 3;
-    let mut rng = StdRng::from_seed([0; 32]);
+    let mut rng = StdRng::from_seed([1; 32]);
```

Comment:
Why did we change that? to distinguish between another key?

Reply:
Yes, I just want to distinct them so we don't accidently run into the same errors.